test/test-utils.ts

```diff
@@ -1,11 +1,11 @@
-/* eslint-disable no-console */
-const fs = require('fs');
-const path = require('path');
-const process = require('process');
+/* eslint-disable no-console, @typescript-eslint/no-non-null-assertion */
+import fs = require('fs');
```

Comment:
please use `import and from`  style imports

Reply:
Done.